cdc/sink/ddlsink/mq/ddlproducer/pulsar_ddl_producer.go

```diff
@@ -53,10 +55,14 @@ func (p *pulsarProducers) SyncBroadcastMessage(ctx context.Context, topic string
 }
 
 // SyncSendMessage sends a message
-// partitionNum is not used,pulsar consume all partitions
+// partitionNum is not used, pulsar consume all partitions
 func (p *pulsarProducers) SyncSendMessage(ctx context.Context, topic string,
 	partitionNum int32, message *common.Message,
 ) error {
+	startTime := time.Now()
+	defer func() {
+		log.Info("pulsarProducers SyncSendMessage cost time", zap.Duration("cost", time.Since(startTime)))
```

Comment:
changefeedID should always be output to the log, which greatly reduces the difficulty of debugging